wheel/generate_type_stubs.py

```diff
@@ -324,51 +340,73 @@ def derive_child_pk_unhardened(pk: G1Element, index: int) -> G1Element: ...
 """
     )
 
-    print_class(f, "G1Element", [], [
-        "SIZE: ClassVar[int] = ...",
-        "def __new__(cls) -> G1Element: ...",
-        "def get_fingerprint(self) -> int: ...",
-        "def pair(self, other: G2Element) -> GTElement: ...",
-        "@staticmethod",
-        "def from_bytes_unchecked(b: bytes) -> G1Element: ...",
-        "@staticmethod",
-        "def generator() -> G1Element: ...",
-        "def __str__(self) -> str: ...",
-        "def __repr__(self) -> str: ...",
-        "def __add__(self, other: G1Element) -> G1Element: ...",
-        "def __iadd__(self, other: G1Element) -> G1Element: ...",
-    ])
-    print_class(f, "G2Element", [], [
-        "SIZE: ClassVar[int] = ...",
-        "def __new__(cls) -> G2Element: ...",
-        "@staticmethod",
-        "def from_bytes_unchecked(b: bytes) -> G2Element: ...",
-        "def pair(self, other: G1Element) -> GTElement: ...",
-        "@staticmethod",
-        "def generator() -> G2Element: ...",
-        "def __str__(self) -> str: ...",
-        "def __repr__(self) -> str: ...",
-        "def __add__(self, other: G2Element) -> G2Element: ...",
-        "def __iadd__(self, other: G2Element) -> G2Element: ...",
-        ])
-    print_class(f, "GTElement", [], [
-        "SIZE: ClassVar[int] = ...",
-        "@staticmethod",
-        "def from_bytes_unchecked(b: bytes) -> GTElement: ...",
-        "def __str__(self) -> str: ...",
-        "def __repr__(self) -> str: ...",
-        "def __mul__(self, rhs: GTElement) -> GTElement: ...",
-        "def __imul__(self, rhs: GTElement) -> GTElement : ...",
-        ])
-    print_class(f, "PrivateKey", [], [
-        "PRIVATE_KEY_SIZE: ClassVar[int] = ...",
-        "def sign_g2(self, msg: bytes, dst: bytes) -> G2Element: ...",
-        "def get_g1(self) -> G1Element: ...",
-        "def __str__(self) -> str: ...",
-        "def __repr__(self) -> str: ...",
-        ])
-
-    print_class(f, "Spend",
+    print_class(
```

Comment:
was this caused by the new version of black?

Reply:
I don't think the formatter was run on this at all before, but maybe